lib/acpi/acpi.c

```diff
@@ -203,7 +203,7 @@ static ACPI_STATUS dev_resource_enum_callback(ACPI_HANDLE obj_handle, UINT32 lev
 	ACPI_STATUS status;
 	ACPI_DEVICE_INFO *dev_info;
 
-	LOG_DBG("%s %p\n", __func__, node);
+	LOG_DBG("%s %p", __func__, node);
```

Comment:
Do we need `__func__` ?

Reply:
> Do we need `__func__` ?

Probably not, as that can be enabled in the log subsystem too. I'll leave it as an exercise to be fixed in a separate PR however (already enough commits in this one).